lib/include/openamp/virtio.h

```diff
@@ -499,6 +511,51 @@ static inline int virtio_reset_device(struct virtio_device *vdev)
 	return 0;
 }
 
+/**
+ * @brief Allocate buffer from the virtio device
+ *
+ * @param vdev	Pointer to virtio device structure.
+ * @param buf	Pointer to the allocated buffer.
```

Comment:
Today we are working at virtio device level with virtual address.  could be nice to indicate it in comment
`Pointer to the allocated buffer (virtual address)`

Reply:
Done